### PR TITLE
Handle epic roadmap for basic process

### PR DESCRIPTION
**Pull request by pranotipr**
Show error message on Epic Roadmap when process is Basic

### CODE REVIEW

File: src/EpicRoadmap/react/Components/EpicRoadmapView.tsx [hunk 1]
```diff
@@ -54,6 +54,14 @@ class EpicRoadmapViewContent extends React.Component<IEpicRoadmapViewProps, IEpi
             return (
                 <Spinner size={SpinnerSize.large} className="loading-indicator" label="Loading..." />
             );
+        } else if (uiState === UIStatus.Error) {
+            return (
+                <MessageBar
+                    messageBarType={MessageBarType.error}
+                    isMultiline={false}
+                >{"Epic Roadmap is not supported for Basic process."}
```
Comment (shutchings) [inline src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: Is this basic process the only thing that could cause a UIStatus.Error? Seems like we'd want to choose a more specific type like the other UIStatus checks are doing below. Something like UIStatus.BasicProcess.

Comment (navneet-g) [reply src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: We should not depend if the process is Basic or not ... we just need to detect if there are less than 4 levels of backlogs (e.g. Taskbacklog, Requirement Backlog and atleast two portfolio backlogs).

Comment (navneet-g) [inline src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: Please update the message to say that Epic Roadmap requires atleast two levels of Portfolio backlogs, and also link to the page which describes portfolio backlogs.

Comment (shutchings) [reply src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: Agreed.

File: src/EpicRoadmap/react/Components/EpicRoadmapView.tsx [hunk 2]
```diff
@@ -54,6 +54,14 @@ class EpicRoadmapViewContent extends React.Component<IEpicRoadmapViewProps, IEpi
             return (
                 <Spinner size={SpinnerSize.large} className="loading-indicator" label="Loading..." />
             );
+        } else if (uiState === UIStatus.Error) {
```
Comment (shutchings) [inline src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: This case still is too generic. Like @navneet-g said we need to detect the levels of backlog and then show this message. It should be similar to the case UIStatus.NoTeamIterations below.

File: src/EpicRoadmap/redux/sagas/FetchAllMetadata.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ export function* FetchAllMetadata() {
     const backlogConfiguration: BacklogConfiguration = yield select(backlogConfigurationForProjectSelector);
 
     if (backlogConfiguration.portfolioBacklogs.length < 2) {
-        yield put(ProgressAwareActionCreator.setError(new Error("Epics backlog level is not configured.")));
+        yield put(ProgressAwareActionCreator.setError(new Error("Epic Roadmap requires atleast two levels of Portfolio Backlogs.")));
```
Comment (shutchings) [inline src/EpicRoadmap/redux/sagas/FetchAllMetadata.ts]: at least is two words. This is more clear I think:

"Epic Roadmap requires at least two portfolio backlog levels."

File: src/EpicRoadmap/react/Components/EpicRoadmapView.tsx (hunk 1 above)
Comment (pranotipr) [reply src/EpicRoadmap/react/Components/EpicRoadmapView.tsx]: As discussed offline, I am not including the link as of now as there is no doc available describing portfolio backlogs correctly.

**Pull request merged**